scripts/bytecodecompare/prepare_report.py

```diff
@@ -224,6 +231,12 @@ def prepare_compiler_input(
 
     settings = CompilerSettings.from_preset(preset)
 
+    command_line = []
+    if execution_arch == ExecutionArchitecture.X86_64.value:
+        command_line = ["/usr/bin/arch", "-64", "-x86_64"]
```

Comment:
What exactly is `-64` here? Switching between the architectures is done via `arch -arm64` and `arch -x86_64` from what I can see.

Reply:
youre right! the arch name should be sufficient

Reply:
turned out that only arch name is not sufficient:
```
COMPILER STDERR:
arch: posix_spawnp: /Users/distiller/project/build/solc/solc: Bad CPU type in executable
```